src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -385,3 +441,58 @@ It corresponds to the extension `ℚ(√-2)/ℚ`. -/
 end quad_char_mod_p
 
 end zmod
+
+/-!
+### Sepcial values of the quadratic character
```

Comment:
```suggestion
### Special values of the quadratic character
```

Reply:
OK. My typing has deteriorated over the last decades...